packages/services/storage/src/schema-change-model.ts

```diff
@@ -197,15 +197,19 @@ export function implement<Model = never>() {
   };
 }
 
+function replaceQuotes(value: string): string {
+  return value.replaceAll("'", '&apos;').replaceAll('"', '&quot;');
+}
+
 export const FieldArgumentDescriptionChangedModel =
   implement<FieldArgumentDescriptionChangedChange>().with({
     type: FieldArgumentDescriptionChangedLiteral,
     meta: z.object({
       typeName: z.string(),
       fieldName: z.string(),
       argumentName: z.string(),
-      oldDescription: z.union([z.string(), z.null()]),
-      newDescription: z.union([z.string(), z.null()]),
+      oldDescription: z.union([z.string().transform(replaceQuotes), z.null()]),
+      newDescription: z.union([z.string().transform(replaceQuotes), z.null()]),
```

Comment:
I only transform `old` and `new` descriptions, I think we don't need transform other descriptions, like https://github.com/kamilkisiela/graphql-hive/blob/3c53032966f2ea7e63f40d22f32e463a4e8fe082/packages/services/storage/src/schema-change-model.ts#L549

Reply:
How does this affect the output of the CLI, Slack and GitHub actions? We also use this information from there, not only from the web client.